src/Components/Web.Extensions/src/Microsoft.AspNetCore.Components.Web.Extensions.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <Description>A medley of Blazor componets for the web.</Description>
```

Comment:
Sorry, I should have made it clear that I was being slightly facetious. I don't think we want to use the term medley in our  description 😨

Reply:
Also typo in `componets`

Reply:
I learned a new word :). collection should be fine.